guides/source/caching_with_rails.md

```diff
@@ -103,6 +115,29 @@ If you want to cache a fragment under certain conditions, you can use
 <% end %>
 ```
 
+#### Collection caching
+
+The `render` helper allows you to do caching for a collection so that it can
+automatically use any cached renders for a template by reading them at once
+instead of one by one like the previous example with `each` did. For instance,
+for a collection of products:
```

Comment:
I'd like to suggest a revision of this. What do you think? :smile:

```
The `render` helper can also cache individual templates rendered for a collection. It can even one up the previous example with `each` by reading all cache templates at once instead of one by one. This is done automatically if the template rendered by the collection includes `cache` call. Take a collection that renders a `products/_product.html.erb` partial for each element:

render products

If `products/_product.html.erb` starts with a `cache` call like so:

<% cache product do %>
  <%= product.name %>
<% end %>

All the cached templates from previous renders will be fetched at once with much greater speed. There's more info on how to make your templates [eligible for collection caching](http://api.rubyonrails.org/classes/ActionView/Template/Handlers/ERB.html#method-i-resource_cache_call_pattern).
```